adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -131,6 +135,25 @@ def _get_required_key(data: Dict[str, Any], name: str) -> Any:
         raise FalParseError("Missing required key: " + name)
     return data[name]
 
+def _parse_remote_config(config: Dict[str, Any]) -> Dict[str, Any]:
+    REMOTE_TYPES_DICT = {
+        "venv": "virtualenv",
+        "conda": "conda"
+    }
+
+    assert config.get("remote_type"), "remote_type needs to be specified."
+
+    remote_type = REMOTE_TYPES_DICT.get(config["remote_type"])
+
+    assert remote_type, f"{config['remote_type']} not recognised. Available remote types: {list(REMOTE_TYPES_DICT.keys())}"
+
+    return {
+        "host": config.get("host"),
+        "target_environment_kind": remote_type,
+        "target_environment_config": {
+            'requirements': config.get('requirements', []),
+        }
```

Comment:
If it were to be conda, then this has to be packages. I think we can simply use all the remaining parameters from the config here (except kind / host etc.)